machinelearninghep.sh

```diff
@@ -0,0 +1,44 @@
+package: MachineLearningHEP
+version: "%(tag_basename)s"
+tag: "run3"
+source: https://github.com/alisw/MachineLearningHEP
+requires:
+  - Python-modules
+prepend_path:
+  PYTHONPATH: ${MACHINELEARNINGHEP_ROOT}/lib/python/site-packages
+---
+#!/bin/bash -e
+
+PIPOPTIONS=""
+
+DEVEL_SOURCES="`readlink $SOURCEDIR || echo $SOURCEDIR`"
+[[ "$DEVEL_SOURCES" != "$SOURCEDIR" ]] && PIPOPTIONS+=" -e"
+
+env -u VIRTUAL_ENV ALIBUILD=1 \
+    python3 -m pip install --force-reinstall \
+    --target="$INSTALLROOT/lib/python/site-packages" \
+    ${PIPOPTIONS} "file://$SOURCEDIR"
+
+# rm -rf "${INSTALLROOT:?}/bin"
+mv "$INSTALLROOT/lib/python/site-packages/bin" "$INSTALLROOT/bin"
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
```

Comment:
You can probably use `alibuild-generate-module --bin` (needs `alibuild-recipe-tools` as `build_requires`) to generate the modulefile (and then append to it the missing parts like PYTHON_PATH).